### PR TITLE
chore: fix typos, remove extra spaces

### PR DESCRIPTION
**Pull request by euprunin**

### CODE REVIEW

File: Manual/Tactics.lean [hunk 1]
```diff
@@ -108,7 +108,7 @@ intro n k
 ```
 ::::
 
-The {tactic}`case` and {tactic}`case'` tactics can be used to select a new main goal using the desired  goal's name.
+The {tactic}`case` and {tactic}`case'` tactics can be used to select a new main goal using the desired goal's name.
```
Comment (david-christiansen) [inline Manual/Tactics.lean:111]: I can't even see the change here in the GH interface. What changed?

**Comment by david-christiansen** [comment]
Thanks!

This looks like it was automatically generated - in particular, it's "correcting" a US spelling to a UK spelling in a vendored JS dependency. What script did you use?

Can you please remove the changes to the vendored JS dependencies? Those should stay identical to the upstream releases.

The changes to the manual itself look useful.

File: Manual/Tactics.lean (hunk 1 above)
Comment (euprunin) [reply Manual/Tactics.lean:111]: I think you have to scroll to the right: `desired  goal` (two spaces) is changed to `desired goal` (one space).

Comment (david-christiansen) [reply Manual/Tactics.lean:111]: Yep, now I see it :)

**Comment by david-christiansen** [comment]
Yes, much better!

Did you make this with a script? If so, it would be nice to add it to CI as a check.

**Comment by david-christiansen** [comment]
The concrete changes are just fine, I simply like to include the thing that prevents future instances of a mistake along with the fixes to the mistakes.

Codespell seems handy, I'll look at incorporating it. What about the double-space script that you used?

**Pull request merged**